<ui version="4.0" >
 <class>newConnectionWidget</class>
 <widget class="QWidget" name="newConnectionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="hostnameLabel" >
       <property name="text" >
        <string>Hostname:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="hostnameLineEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="portLabel" >
       <property name="text" >
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="portLineEdit" />
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>77</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
